### ISSUE TITLE
NPE in TextField.setString()

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Perundurai-Products/microemu#7]
```
According to the MIDP API at
http://java.sun.com/javame/reference/apis/jsr118/javax/microedition/lcdui/T
extField.html#setString(java.lang.String), passing null to
TextField.setString() should empty the field. Instead, a
NullPointerException is thrown (there is a missing null check in the
overloaded method).

```

Original issue reported on code.google.com by `bar...@gmail.com` on 18 Nov 2009 at 11:12

**Comment by GoogleCodeExporter** [comment]
```
Here's a patch that fixes it.
```

Original comment by `amichair` on 3 Jan 2010 at 2:11

Attachments:
- [fix_textfield_null.diff](https://storage.googleapis.com/google-code-attachments/microemu/issue-7/comment-1/fix_textfield_null.diff)

**Comment by GoogleCodeExporter** [comment]
```
Fixed in trunk (future version 3.0)
```

Original comment by `bar...@gmail.com` on 6 Jan 2010 at 9:42
- Changed state: **Fixed**
- Added labels: **Milestone-3.0**
